tests/installation/upgrade_select_opensuse.pm

```diff
@@ -56,6 +56,13 @@ sub run {
         record_soft_failure 'installed product incompatible';
     }
 
+    if (check_screen('inst-betawarning')) {
```

Comment:
Let's use multitag match here not to wait for beta warning when it's not there. 
And then apply logic using match_has_tag. Or it's not clear if we should expect it? I'm also not sure why we cannot do screen assertion here. And if beta warning is shown when it's not beta, we should file a bug for that instead of masking it with soft-failure. WDYT?

Reply:
:+1:

Reply:
In the issue it sounds like its not sure if there should be a second beta warning so assert_screen could fail where it shouldn't.
The soft_fail mechanism was already there, just duplicated it